### ISSUE TITLE
Delete old Twilio number

### ISSUE THREAD

**Issue by lamby** [takeyourmeds/takeyourmeds-web#85]
```
>>> from takeyourmeds.telephony.utils import *
>>> send_sms('07751237519', 'test 1')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/usr/share/python/takeyourmeds/local/lib/python2.7/site-packages/takeyourmeds/telephony/utils.py", line 16, in send_sms
    from_=settings.TWILIO_FROM,
  File "/usr/share/python/takeyourmeds/local/lib/python2.7/site-packages/twilio/rest/resources/messages.py", line 122, in create
    return self.create_instance(kwargs)
  File "/usr/share/python/takeyourmeds/local/lib/python2.7/site-packages/twilio/rest/resources/base.py", line 365, in create_instance
    data=transform_params(body))
  File "/usr/share/python/takeyourmeds/local/lib/python2.7/site-packages/twilio/rest/resources/base.py", line 200, in request
    resp = make_twilio_request(method, uri, auth=self.auth, **kwargs)
  File "/usr/share/python/takeyourmeds/local/lib/python2.7/site-packages/twilio/rest/resources/base.py", line 164, in make_twilio_request
    uri=resp.url, msg=message, code=code)
TwilioRestException: 
HTTP Error Your request was:

POST https://api.twilio.com/2010-04-01/Accounts/AC7d6b676d2a17527a71a2bb41301b5e6f/Messages.json

Twilio returned the following information:

The number +447751237519 is unverified. Trial accounts cannot send messages to unverified numbers; verify +447751237519 at twilio.com/user/account/phone-numbers/verified, or purchase a Twilio number to send messages to unverified numbers.

More information may be available here:

https://www.twilio.com/docs/errors/21608

```

**Comment by lamby** [comment]
ie. it needs feeding some money

**Comment by lamby** [comment]
Then assign back please :)

**Comment by DeckOfPandas** [comment]
Assigning back to @lamby 

I've upgraded to a paid-for account, and have also purchased a new phone number: a local number that can send and receive voice and SMS and ends with TYM (896). 

Does anything need changing in order to use this new number?

Let's delete the old number (0114 303 2046) when possible to save me the $1/month!

**Comment by DeckOfPandas** [comment]
I might have done a bad...

I deleted the old number as I thought I was supposed to from this ticket still being open. Since then, voice calls are logged as "failed":

![screenshot](https://cloud.githubusercontent.com/assets/2399432/11583423/79223016-9a51-11e5-9e2b-35aeb749d85c.png)

**Comment by fiona-stacey** [comment]
Hi Helen i tried to set up a text alert yesterday but didn't get a text. I couldn't set up any particular message and, as I now assume there are some sticky bits, what's the best way to record your message? 
Fi

Sent from Samsung Mobile

<div>-------- Original message --------</div><div>From: Helen Jackson notifications@github.com </div><div>Date:04/12/2015  06:37  (GMT+00:00) </div><div>To: takeyourmeds/takeyourmeds-web takeyourmeds-web@noreply.github.com </div><div>Subject: Re: [takeyourmeds-web] Delete old Twilio number (#85) </div><div>
</div>I might have done a bad...

I deleted the old number as I thought I was supposed to from this ticket still being open. Since then, voice calls are logged as "failed":

**Comment by DeckOfPandas** [comment]
@fiona-stacey : I'll get this sorted out today and will confirm when it's fixed! :)

**Comment by lamby** [comment]
Fixed in #151
